lib/api/src/modules/url.ts

```diff
@@ -178,6 +183,14 @@ export const init: ModuleFn = ({ store, navigate, state, provider, fullAPI, ...r
       }
     });
 
+    fullAPI.on(GLOBALS_UPDATED, ({ globals, defaultGlobals, initialGlobals }) => {
```

Comment:
I'm not sure why we store and emit the `defaultGlobals` and `initialGlobals` separately. They sort of seem like the same thing, coming from different places, and they are treated the same way one the manager side too, right?

Reply:
Sure. That's fair. I integrated `defaultGlobals` into `initialGlobals` now, so we only have to pass that.